ui/src/user/components/SignUpPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Row, Card, Input, Button } from 'antd';
+import { Field, Form, Formik } from 'formik';
+import DocumentHead from '../../common/components/DocumentHead';
+
+function renderFormInput({ field, form, ...props }) {
+  return <Input {...field} {...props} />;
+}
+
+function SignUpPage({ onSignUpClick }) {
```

Comment:
you can have a look at `SubscribeJobsForm` as an example which is better than `LocalLogin` one (using `TextField` and validation) since it's user facing one.